docs/reqs/reqs.tex

```diff
@@ -231,24 +231,24 @@ \subsection{Non-RPAS Air-Ground I/O}
 
 \subsection{Computing Platform}
 \begin{enumerate}[label=F.CP.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
+	\item Computing Platform must use noise-tolerant communications protocols to avoid drone interference.
 	\item Computing Platform must contain at least 1MB of non-volatile memory to store machine learning weights.
-	\item Interface with the FPGA must be capable of bitrates sufficient for video streaming (>1.5Mbps)
+	\item Interface with the FPGA must be capable of bitrates sufficient for video streaming (>1.5Mbps).
```

Comment:
Be consistent with numerical formatting. (https://www.nist.gov/pml/weights-and-measures/writing-metric-units)

```suggestion
	\item Interface with the FPGA must be capable of bitrates sufficient for video streaming (>1.5 Mbps).
```